docs/reference/environment-variables.md

```
@@ -345,6 +345,61 @@ export HOLMES_PASSTHROUGH_BLOCKED_HEADERS="authorization,cookie,set-cookie,x-int
+
+**Example:**
+```bash
+export HOLMES_K8S_AUTH_MODE="request_token"
+```
+
+### HOLMES_K8S_REQUEST_TOKEN_HEADERS
+**Default:** `"X-Auth-Request-Id-Token,Authorization"`
+
+Ordered, comma-separated list of headers to read the user's token from when
+`HOLMES_K8S_AUTH_MODE=request_token`. The first non-empty match wins, and a
+leading `Bearer ` prefix is stripped. `oauth2-proxy` injects
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)